cmd/util/cmd/execution-state-extract/cmd.go

```diff
@@ -378,10 +378,10 @@ func run(*cobra.Command, []string) {
 	switch chainID {
 	case flow.Emulator:
 		burnerContractChange = migrations.BurnerContractChangeDeploy
-		evmContractChange = migrations.EVMContractChangeDeploy
+		evmContractChange = migrations.EVMContractChangeDeployMinimalAndUpdateFull
 	case flow.Testnet, flow.Mainnet:
```

Comment:
Do we need to handle PreviewNet or that's outside of the scope of this pr and we do it traditional way?

Reply:
Previewnet is not migrated, so need to be handled separately / manually. That is also what @janezpodhostnik mentioned in the Discord thread (see link above)